vita3k/gxm/include/gxm/types.h

```diff
@@ -1427,9 +1427,18 @@ enum SceGxmFragmentProgramInputs : int {
     _SCE_GXM_FRAGMENT_PROGRAM_INPUT_LAST = 1 << 15
 };
 
-enum SceGxmSpecialFlag {
-    SCE_GXM_SPECIAL_HAS_INDEX_SEMANTIC = 1 << 1,
-    SCE_GXM_SPECIAL_HAS_INSTANCE_SEMANTIC = 1 << 2
+enum SceGxmProgramFlags : std::uint32_t {
+    SCE_GXM_PROGRAM_FLAG_PER_INSTANCE_MODE = 1 << 1,
+    SCE_GXM_PROGRAM_FLAG_DISCARD_USED = 1 << 3,
+    SCE_GXM_PROGRAM_FLAG_DEPTH_USED = 1 << 4,
+    SCE_GXM_PROGRAM_FLAG_SPRITECOORD_USED = 1 << 5,
+    SCE_GXM_PROGRAM_FLAG_NATIVECOLOR_USED = 1 << 6,
+    SCE_GXM_PROGRAM_FLAG_FRAGCOLOR_USED = 1 << 7,
+    SCE_GXM_PROGRAM_FLAG_NATIVECOLOR_MSAA2X = 1 << 8,
+    SCE_GXM_PROGRAM_FLAG_NATIVECOLOR_MSAA4X = 1 << 9,
+    SCE_GXM_PROGRAM_FLAG_OUTPUT_UNDEFINED = 1 << 16,
+    SCE_GXM_PROGRAM_FLAG_INDEX_USED = 1 << 17,
```

Comment:
Very unsure about this, need testing to see if anything break. There are games that use this, and I'm not in remember of how these values work.

Reply:
VitaGL's implementation of glClear still works, and it uses the INDEX semantic. So it's fine.

Reply:
Yeah that's true. I don't think anyone is available to test further more, so I will assume it all goes right.